### PR TITLE
Fixing some compiler warnings.

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

**Comment by nmittler** [comment]
@ejona86 should be an easy one :)

File: netty/build.gradle [hunk 1]
```diff
@@ -13,4 +13,4 @@ test {
     jvmArgs "-Xbootclasspath/p:" + configurations.alpnboot.asPath
 }
 
-javadoc.options.links 'https://netty.io/4.1/api/'
+javadoc.options.links 'http://netty.io/4.1/api/'
```
Comment (ejona86) [inline netty/build.gradle]: This started happening because the https web site went down. I would like to swap back to https if it starts working again.

**Comment by ejona86** [comment]
@nmittler LGTM

**Comment by nmittler** [comment]
@ejona86 SGTM

Cherry-picked as a6c51e7

**Pull request closed without merging**